libPhoneNumberGeocoding/NBPhoneNumberOfflineGeocoder.m

```diff
@@ -0,0 +1,169 @@
+//
+//  NBPhoneNumberOfflineGeocoder.m
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBPhoneNumberOfflineGeocoder.h"
+#import "Metadata/NBGeocoderMetadataHelper.h"
+#import "NBPhoneNumber.h"
+#import "NBPhoneNumberUtil.h"
+
+@implementation NBPhoneNumberOfflineGeocoder {
+ @private
+  NBPhoneNumberUtil *_phoneNumberUtil;
+  NSCache<NSString *, NBGeocoderMetadataHelper *> *_metadataHelpers;
+  NSNumber const *_US_REGION_CODE;
+  NBGeocoderMetadataHelperFactory _metadataHelperFactory;
+}
+
+NSString *const _INVALID_REGION_CODE = @"ZZ";
```

Comment:
Make this static, and rename it to remove the _ prefix

Reply:
Done.

Reply:
It's still not static, and still has a _ prefix